rct229/rules/section15.py

```diff
@@ -274,7 +274,7 @@ def get_calc_vals(self, context, data=None):
 
             return {
                 "user_transformer_efficiency": context.user["efficiency"],
-                "required_user_transformer_min_efficiency": table_8_4_4_eff(
+                "required_user_transformer_min_efficiency": table_8_4_4_lookup(
```

Comment:
Needs to be modified since the return value for  table_8_4_4_lookup() has changed.